ext-php-rs-derive/src/method.rs

```diff
@@ -92,8 +93,14 @@ pub fn parser(input: &mut ImplItemMethod) -> Result<(TokenStream, Method)> {
         }
     };
 
+    let name = if ident == "__construct" {
```

Comment:
We should apply this to all magic methods.

Reply:
Oh that's a good point. How would we handle `__toString()`?

Should we just strip the double underscore prefix, normalize the token, then reattach it? In which case `__to_string` would become `__toString`.

Reply:
Have you tried to to just pass `__to_string` to `ident_case` to see how it goes ?

Reply:
Right, sorry. Should have explained that the reason for '__construct' special case is it got converted to 'construct'. I t doesn't seem like `indent_case` respects leading underscores.